Framework/PythonInterface/test/python/mantid/kernel/ConfigObserverTest.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import sys
+from mantid.kernel import ConfigService, ConfigObserver
+
+if sys.version_info.major == 3:
```

Comment:
Can we swap this around and check for 2 first and then fallback: https://astrofrog.github.io/blog/2016/01/12/stop-writing-python-4-incompatible-code/